src/main/java/Main.java

```diff
@@ -0,0 +1,96 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) throws DukeException{
```

Comment:
I feel like a lot of the code in Main can be better abstracted out to its other classes, it feels cluttered and some of the code being executed aren't really Main's responsibility.  

[`Single Responsibility Principle`](https://hackernoon.com/you-dont-understand-the-single-responsibility-principle-abfdd005b137)